lib/features/stacking/index.js

```diff
@@ -21,29 +21,20 @@ export const position = {
         ) || []
     ).join('-'),
 
-  getBorderRadii: ({ top, bottom, square }, radius) => (
-    top
-      ? {
-        borderBottomLeftRadius: '0px',
-        borderBottomRightRadius: '0px',
-        borderTopLeftRadius: radius,
-        borderTopRightRadius: radius
-      }
-      : bottom
-        ? {
-          borderBottomLeftRadius: radius,
-          borderBottomRightRadius: radius,
-          borderTopLeftRadius: '0px',
-          borderTopRightRadius: '0px'
-        }
-        : square
-          ? {
-            borderRadius: '0px'
-          }
-          : {
-            borderRadius: radius
-          }
-  ),
+  getBorderRadii: ({ top, bottom, left, right, square }, radius) => {
```

Comment:
This looks great. Except maybe coming up with some higher kinded polymorphism in the category of bifunctors, I can imagine a clean solution to this problem. And then it wouldn't be understandable anyway. This is about the best I can imagine.